src/Microsoft.ML.Ensemble/OutputCombiners/RegressionStacking.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+using System;
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.CommandLine;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.Ensemble.OutputCombiners;
+using Microsoft.ML.Runtime.EntryPoints;
+using Microsoft.ML.Runtime.Model;
+
+[assembly: LoadableClass(typeof(RegressionStacking), typeof(RegressionStacking.Arguments), typeof(SignatureCombiner),
+    Stacking.UserName, RegressionStacking.LoadName)]
+[assembly: LoadableClass(typeof(RegressionStacking), null, typeof(SignatureLoadModel),
+    Stacking.UserName, RegressionStacking.LoadName)]
+
+namespace Microsoft.ML.Runtime.Ensemble.OutputCombiners
+{
+    using TScalarPredictor = IPredictorProducing<Single>;
+    public sealed class RegressionStacking : BaseScalarStacking<SignatureRegressorTrainer>, IRegressionOutputCombiner, ICanSaveModel
+    {
+        public const string LoadName = "RegressionStacking";
+        public const string LoaderSignature = "RegressionStacking";
+
+        private static VersionInfo GetVersionInfo()
+        {
+            return new VersionInfo(
+                modelSignature: "RSTACK C",
+                verWrittenCur: 0x00010001, // Initial
+                verReadableCur: 0x00010001,
+                verWeCanReadBack: 0x00010001,
+                loaderSignature: LoaderSignature);
```

Comment:
>LoaderSignature [](start = 33, length = 15)

I'm not sure how you want to do it, but this loader signature *must* be identical to that declared in the assembly level attribute, so it should ideally use the same variable name.

Reply:
To be clear: I am talking about this line.

```
[assembly: LoadableClass(typeof(RegressionStacking), null, typeof(SignatureLoadModel),
    Stacking.UserName, RegressionStacking.LoadName)]
```

That `LoadName` *must* be the same as the thing here, since that is controls whether the load actually succeeds or not.

I was *not* referring to your change on line 12, which has to do with how things are instantiated from the command line.

---
In reply to: [197500796](https://github.com/dotnet/machinelearning/pull/379#discussion_r197500796) [](ancestors = 197500796)

Reply:
my bad, SignatureLoaderModule should have LoaderSignature, not SignatureCombiner

---
In reply to: [197999644](https://github.com/dotnet/machinelearning/pull/379#discussion_r197999644) [](ancestors = 197999644,197500796)